File: exercism_data/src/333.rb
class Phrase
  def initialize(phrase)
    @phrase = phrase
  end

  def word_count
    @phrase.strip
           .downcase
           .gsub(/[^a-zA-Z0-9']/, " ")
           .split(" ")
           .each_with_object(Hash.new(0)) { |word, hash| hash[word] += 1 }
  end
end

File: assignments/python/wc/src/333.py
def word_count(text):
	text = text.split()
	list_of_words = {}
	for i in text:
		if i in list_of_words:
			list_of_words[i] += 1
		else:
			list_of_words[i] = 1
	print list_of_words
	return list_of_words
